go/pkg/client/tree.go

```diff
@@ -464,12 +464,16 @@ func getRelPath(base, path string) (string, error) {
 // ComputeOutputsToUpload transforms the provided local output paths into uploadable Chunkers.
 // The paths have to be relative to execRoot.
 // It also populates the remote ActionResult, packaging output directories as trees where required.
-func (c *Client) ComputeOutputsToUpload(execRoot string, paths []string, cache filemetadata.Cache) (map[digest.Digest]*uploadinfo.Entry, *repb.ActionResult, error) {
+func (c *Client) ComputeOutputsToUpload(execRoot, workingDir string, paths []string, cache filemetadata.Cache) (map[digest.Digest]*uploadinfo.Entry, *repb.ActionResult, error) {
```

Comment:
Could you keep old interface for gradual migration in our repository?

Reply:
Is this still relevant? Can you elaborate on what gradual migration you're thinking of?